earth_enterprise/docs/landing_page/manual/answer/6004748.html

```diff
@@ -286,7 +286,7 @@
 </td>
 <td>Localize international URL for support answers
 </td>
-<td><code>https://support.google.com/earth/#topic=4363013</code>
```

Comment:
There are other configuration values on this page that point to google.com domains (like www.google.com and earth.google.com).  Can we change those to reference opengee.org or something on the github wiki?